src/main/java/mate/jdbc/service/impl/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.ManufacturerService;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return null;
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        return null;
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        return null;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        return null;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return false;
+    }
+}
```

Comment:
I think you forgot to write implementation of these methods

Reply:
![image](https://user-images.githubusercontent.com/111144285/201768420-5746318c-ecc2-4770-9bbb-c91d7aebc5c6.png)